src/lib/openapi/spec/ready-check-schema.ts

```diff
@@ -0,0 +1,22 @@
+import type { FromSchema } from 'json-schema-to-ts';
+
+export const readyCheckSchema = {
```

Comment:
Something feels off to me here. Glancing at our API docs, this will only manifest for 200 responses. We can't ever send "BAD" on a 200 response though, the 500 tag will get it's own schema, which I don't think will include "BAD"

Maybe not an issue, since this is unlikely to be used a lot by open api generators, just feels funny

Reply:
Yes, good call, I think it's same with the health endpoint. Let me address that